docs/anchoring-assets/anchor-platform/sep31/integration.mdx

```diff
@@ -15,6 +18,258 @@ The following may also be required depending on your use case:
+
+## Fetching Transaction Status Updates
+
+To facilitate cross-border payments, you'll need to be able to detect when a sending organization has sent your business an on-chain payment and determine which transaction that payment was meant to fulfil.
+
+The easiest way to do that is to run the Stellar Observer, which will detect these payments and update the corresponding transaction record with information about the payment. Your business can then detect these updates by polling the `GET /transactions` Platform API endpoint.
+
+### Running the Stellar Observer
+
+The Stellar Observer monitors the Stellar ledger for payments made to your account(s) and updates the corresponding transaction records with on-chain payment information. To run the observer, add the following to your docker compose file.
+
+<CodeExample>
+
+```yaml
+services:
+  ...
+  observer:
+    image: stellar/anchor-platform:2.0.0-rc.2
+    command: --stellar-observer
+    env_file:
+      - ./dev.env
+    volumes:
+      - ./config:/home
+```
+
+</CodeExample>
+
+### Polling for Received Payments
+
+The Stellar Observer makes `PATCH /transactions` requests to the Platform API whenever it detects payments received for transactions initiated by sending organizations, updating the transaction's `transfer_received_at` date-time. Your business should periodically poll the `GET /transactions` Platform API endpoint to detect these updates like so.
+
+<CodeExample>
+
+```bash
+curl http://localhost:8080/transactions?sep=31&order_by=transfer_received_at&order=desc
+```
+
+</CodeExample>
+
+The response will include a list of cross-border payment transactions initiated by sending organizations ordered by when a payment was received for that transaction. For each transaction returned, your business should check whether or not it has already detected the payment for that transaction. If it has, you have detected all payments made to your account(s).
+
+## Providing Transaction Status Updates
+
+Once your business detects that it has received an on-chain payment for a specific transaction, it must make the off-chain payment to the recipient and update the transaction's status to `completed` via a [`PATCH /transactions`][patch-transactions] request.
+
+<CodeExample>
+
+```json
+{
+  "transaction": {
+    "id": "<id>",
+    "status": "completed",
+    "message": "a deposit to the recipient's bank has been initiated"
+  }
+}
+```
+
+</CodeExample>
+
+## Fee Callback Endpoint
+
+Your business may want to offer sending organizations the option to skip the quote creation process, allowing your business to use a rate determined at the time funds are paid out to the recipient. In this case, the Anchor Platform will not make a `GET /rate` request, but you will still need to provide the fee your business will charge for these types of transactions using the [`GET /fee`][get-fee] endpoint.
```

Comment:
We say "may want to offer", but there are no explicit way to prohibit it, right? 
If they don't implement GET /fee it means they do not offer such functionality. As the result, /transaction endpoint will simply return 500 because trying to get a callback to /fee will throw.
Is my understanding correct?

Reply:
The Anchor Platform shouldn't even make the `GET /fee` request to the business if `quotes_supported` is not `true`. So the Anchor Platform should return 400 if they try not to pass a quote.

Reply:
You mean quotes_required?

Reply:
Yes my mistake. So if `quotes_required` is true, the anchor platform won't make the `GET /fee` request.